Documentation/client.md

```diff
@@ -79,35 +95,39 @@ If you don't yet have a Rook cluster running, refer to our [Quickstart Guides](R
 2. Create an object storage user
 
    ```bash
-   rook object user create rook-user "A rook rgw User"
+   rook object user create rook-user "my object store user"
    ```
 
-3. Get the connection information for accessing object storage
+### Consume the Object Storage
+Use an S3 compatible client to create a bucket in the object store. If you are running in Kubernetes,
+the s3cmd tool is included in the [Rook toolbox](toolbox.md) pod.
```

Comment:
I think it would be better in this flow to not make the user download the huge toolbox.  The `rook-client` container used to have `s3cmd` in it, but it doesn't anymore.  Is that because `s3cmd` just plain doesn't work on Alpine?  If it works, I think we should keep `s3cmd` in the `rook-client` container and these instructions should not tell the user to use the toolbox.